cpr/cookies.cpp

```diff
@@ -3,7 +3,7 @@
 namespace cpr {
 std::string Cookies::GetEncoded(const CurlHolder& holder) const {
     std::stringstream stream;
-    for (const std::pair<std::string, std::string>& item : map_) {
+    for (const auto& item : map_) {
```

Comment:
Since I hate `auto`, how about this:
```suggestion
    for (const std::pair<const std::string, std::string>& item : map_) {
```